Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/IISWebAppDeployV2/externals.json

```diff
@@ -0,0 +1,21 @@
+
+{
+    "nugetv2": {
+        "VstsTaskSdk": {
+            "version": "0.7.1",
```

Comment:
`0.7.1` is quite old version. Do we need to consider the latest version?

Reply:
Latest versions are giving some errors. Have tried all versions: 0.8.2 , 0.10.0 and 0.11.0